ultradns/resource_ultradns_probe_http.go

```diff
@@ -138,9 +141,9 @@ func resourceUltradnsProbeHTTPCreate(d *schema.ResourceData, meta interface{}) e
 
 	uri := resp.Header.Get("Location")
 	d.Set("uri", uri)
-	id := strings.Split(uri,"/probes")[1]
+	id := fmt.Sprintf("%s:%s:%s", d.Get("name"), d.Get("zone"), strings.Split(uri, "probes/")[1])
```

Comment:
Please make sure that these colons separators doesn't collide with any actual colons.

Reply:
yup it won't collide with actual colons as per the RFC there will be no `:` in the zone and resource record names